prep-css-syntax/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Syntax</title>
+  </head>
+  <style>
+    h1 {
+      font-family: Arial, Helvetica, sans-serif;
+    }
+    p {
+      font-size: 2rem;
+    }
+    h3 {
+      color: firebrick;
+    }
+    h2 {
+      font-style: italic;
+    }
+  </style>
```

Comment:
In general, we want to put `<style></style>` element in the `<head></head>` of the document.

Reply:
Updates have been pushed.